src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorContentTypeChangeListener.cs

```diff
@@ -83,6 +84,8 @@ internal class RazorContentTypeChangeListener : ITextBufferContentTypeListener
             _editorOptionsFactory = editorOptionsFactory;
         }
 
+        public TrackingLSPDocumentManager LspDocumentManager => _lspDocumentManager;
```

Comment:
Why the change?

Reply:
Hmm, looks like might've accidentally changed it after all. I was probably trying to apply quick-fix for namespace and somehow got this wrapped into a property? Anyway, got rid of it. 

---
In reply to: [437761586](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437761586) [](ancestors = 437761586,437132381)